rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -903,7 +903,14 @@ def _maybe_decode_erc20_721_transfer(
                     action_item.asset == found_token and
                     action_item.from_event_type == transfer.event_type and
                     action_item.from_event_subtype == transfer.event_subtype and
-                    (action_item.amount is None or action_item.amount == transfer.balance.amount) and  # noqa: E501
+                    (
+                        (action_item.amount is None or action_item.amount == transfer.balance.amount) or  # noqa: E501
+                        (
+                            action_item.amount_error_tolerance is not None and
+                            action_item.amount is not None and
+                            abs(action_item.amount - transfer.balance.amount) < action_item.amount_error_tolerance  # abs used for supporting also different cases than stETH, if they exist # noqa: E501
```

Comment:
no need for the comment. Makes sense to always use abs for tolerance.

```suggestion
                            abs(action_item.amount - transfer.balance.amount) < action_item.amount_error_tolerance
```

Reply:
I left the comment only temporarly for explaining during the review.